### PR TITLE
Add Yews package

### PR DESCRIPTION
**Pull request by lijunzh**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by lijunzh** [comment]
I, @lijunzh, am willing to be listed here as a maintainer of this package.

**Comment by lijunzh** [comment]
@conda-forge/staged-recipes This PR is ready for review.

**Comment by lijunzh** [comment]
@chrisburr Sure.

File: recipes/yews/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:  
+  host:
+    - pip
+    - python>=3.6
```
Comment (chrisburr) [inline recipes/yews/meta.yaml]: You need a space here for some of the conda-forge scripts to work:
```suggestion
    - python >=3.6
```
(and for the other two)

**Comment by lijunzh** [comment]
@chrisburr The comments have now been removed. Feel free to merge it now.

File: recipes/yews/meta.yaml (hunk 1 above)
Comment (lijunzh) [reply recipes/yews/meta.yaml]: Thanks a lot for the suggestion. I did not know that before.

Comment (chrisburr) [reply recipes/yews/meta.yaml]: The other two version constraints need fixing. (IIRC this causes feedstock creation to fail)

Comment (lijunzh) [reply recipes/yews/meta.yaml]: @chrisburr I thought that rule only applies to `host`. They have been fixed now.

**Pull request merged**